### PR TITLE
Set width of editable area to be same as that of the Placeholder

### PR DESCRIPTION
**Pull request by donnapep**
Presentation - http://goo.gl/S0hQRU

### CODE REVIEW

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. Thx.

File: test/integration/custom-font.html [hunk 1]
```diff
@@ -35,8 +35,10 @@
       test("should load a custom font", function() {
         // polyserve testing in browser, change to below
         // assert.equal(document.styleSheets[0].cssRules[17].cssText,
-        assert.equal(document.styleSheets[0].cssRules[6].cssText,
-          '@font-face { font-family: \'My font name\'; src: url("https://my.custom.font/My%20font\'%20name.otf"); }');
+        setTimeout(function() {
+          assert.equal(document.styleSheets[0].cssRules[6].cssText,
+            '@font-face { font-family: \'My font name\'; src: url("https://my.custom.font/My%20font\'%20name.otf"); }');
+        }, 1);
```
Comment (stulees) [inline test/integration/custom-font.html]: @donnapep Can you clarify why only 1 millisecond?

Comment (donnapep) [inline test/integration/custom-font.html]: @stulees It's just to add it to the event queue which is enough time for the font loading to complete.

Comment (stulees) [inline test/integration/custom-font.html]: @donnapep Okay thanks

**Comment by stulees** [comment]
@donnapep 👍

**Pull request merged**